### PR TITLE
O3-1229: Instructions should align with actual display

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
This PR address the input fields used, to show an `optional` text to the fields which are not required to be filled by the user.

## Screenshots

image

## Other

*None.*

### CODE REVIEW

**Review by ibacher — COMMENTED** [review]
Part of me wonders if we should also translate the parentheses, but in principle this looks good. Can you fix the failing test and merge conflict please?

File: packages/esm-patient-registration-app/translations/en.json [hunk 1]
```diff
@@ -43,18 +43,19 @@
...
   "patient": "Patient",
-  "patientNameKnown": "Is Patient's Name Known?",
+  "patientNameKnown": "Patient's Name is Known?",
   "phoneEmailLabelText": "Phone, Email, etc.",
-  "phoneNumberInputLabelText": "Phone number (optional)",
-  "registerPatient": "Create Patient",
+  "phoneNumberInputLabelText": "Phone number",
+  "registerPatient": "Register Patient",
   "registrationSuccessToastDescription": "The patient can now be found by searching for them using their name or ID number",
   "registrationSuccessToastTitle": "New Patient Created",
```
Comment (ZacButko) [inline packages/esm-patient-registration-app/translations/en.json:60]: `registerPatient` changed from "Create Patient" to "Register Patient"... did you also want to update this one to be "New Patient Registered" ?

**Review by ZacButko — APPROVED** [review]
Nice work. Thanks @vinayvenu

File: packages/esm-patient-registration-app/translations/en.json (hunk 1 above)
Comment (vasharma05) [reply packages/esm-patient-registration-app/translations/en.json:60]: No, let me check once again

Comment (vasharma05) [reply packages/esm-patient-registration-app/translations/en.json:60]: Hi @ZacButko!
There are a lot of translations which are to be fixed. I'll bring out a new PR by this week end. This includes providing a default value to all the translation keys.
Thanks!

**Pull request merged**